src/neo/IO/Helper.cs

```diff
@@ -322,6 +322,15 @@ public static void WriteFixedString(this BinaryWriter writer, string value, int
             }
         }
 
+        public static void WriteArray<T>(this BinaryWriter writer, T[] value) where T : class, ISerializable
+        {
+            writer.WriteVarInt(value.Length);
+            foreach (var item in value)
+            {
+                item.Serialize(writer);
+            }
+        }
```

Comment:
https://github.com/neo-project/neo/blob/98ec16958b9ee4e8e84660c88bcd8a1e01be894b/src/neo/IO/Helper.cs#L271-L278

There is a method with the same function, and it should not be written again.